react/Login.js

```diff
@@ -141,9 +141,34 @@ Login.getSchema = () => ({
         }
       }
     },
-    EXPERIMENTAL_enableGoogleOneTap: {
-      title: 'admin/editor.login.enableGoogleOneTap',
-      type: 'boolean'
+    hasGoogleOneTap: {
+      title: 'admin/editor.login.googleOneTap.title',
+      type: 'boolean',
+    },
+  },
+  dependencies: {
+    ...LoginSchema.dependencies,
+    hasGoogleOneTap: {
+      oneOf: [
+        {
+          properties: {
+            hasGoogleOneTap: {
+              enum: [true],
+            },
+            googleOneTapAlignment:{
+              title: 'admin/editor.login.googleOneTap.alignment',
+              type: 'string',
+              enum: [GoogleOneTapAlignment.RIGHT, GoogleOneTapAlignment.LEFT],
+              default: GoogleOneTapAlignment.RIGHT,
+            },
+            googleOneTaMarginTop:{
```

Comment:
Here and everywhere else
```suggestion
            googleOneTapMarginTop:{
```